### ISSUE TITLE
Liquid Exception: wrong number of arguments (1 for 0) in ./main_feed.xml

### ISSUE THREAD

**Issue by maiertech** [octopress/tag-helpers#1]
I did a `bundle update` on my project with the only change being `octopress-tag-helpers` being updated from `1.0.6` to `1.0.7`. When building my site I get the following stack trace:

```
  Liquid Exception: wrong number of arguments (1 for 0) in ./main_feed.xml
rake aborted!
ArgumentError: wrong number of arguments (1 for 0)
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-filters-1.3.3/lib/octopress-filters.rb:11:in `site'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/strainer.rb:43:in `invoke'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/context.rb:82:in `invoke'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/variable.rb:50:in `block in render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/variable.rb:38:in `each'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/variable.rb:38:in `inject'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/variable.rb:38:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-tag-helpers-1.0.7/lib/octopress-tag-helpers/var.rb:44:in `get_value'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-return-tag-1.0.1/lib/octopress-return-tag.rb:17:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:109:in `block in render_all'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:96:in `each'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:96:in `render_all'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:82:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/template.rb:128:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/template.rb:138:in `render!'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-include-tag-1.1.1/lib/octopress-include-tag.rb:46:in `block in render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/context.rb:112:in `stack'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-include-tag-1.1.1/lib/octopress-include-tag.rb:43:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:109:in `block in render_all'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:96:in `each'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:96:in `render_all'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/block.rb:82:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/template.rb:128:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/liquid-2.6.2/lib/liquid/template.rb:138:in `render!'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/convertible.rb:106:in `render_liquid'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/convertible.rb:233:in `do_layout'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-hooks-2.6.1/lib/octopress-hooks.rb:350:in `do_layout'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-tag-helpers-1.0.7/lib/octopress-tag-helpers/hooks.rb:9:in `do_layout'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/page.rb:122:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-ink-1.1.4/lib/octopress-ink/jekyll/page.rb:18:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/site.rb:298:in `block in render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/site.rb:297:in `each'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/site.rb:297:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/octopress-hooks-2.6.1/lib/octopress-hooks.rb:197:in `render'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/site.rb:51:in `process'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/command.rb:28:in `process_site'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/commands/build.rb:56:in `build'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/gems/jekyll-2.5.3/lib/jekyll/commands/build.rb:34:in `process'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/bundler/gems/website-theme-ecdb1c73997c/lib/tasks/build.rake:82:in `block (2 levels) in <top (required)>'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/bundler/gems/website-theme-ecdb1c73997c/lib/tasks/build.rake:90:in `block (2 levels) in <top (required)>'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/bin/ruby_executable_hooks:15:in `eval'
/home/vagrant/.rvm/gems/ruby-2.1.5@website-sdgs/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => site:build
(See full trace by running task with --trace)
```

**Comment by imathis** [comment]
Thanks for the report! I just released a fix in version 1.0.8.

**Comment by maiertech** [comment]
Works! Many thanks!
